app/controllers/admin/enterprises_controller.rb

```diff
@@ -65,6 +65,8 @@ def update
       update_tag_rules(tag_rules_attributes) if tag_rules_attributes.present?
       update_enterprise_notifications
 
+      delete_custom_tab if params[:custom_tab] == 'false'
```

Comment:
Looks like good housekeeping. But I wonder if people may like to edit the tab without making it public yet. Maybe they have a draft and then want to publish it later?

I would propose a `display` attribute on the custom_tab model. Then you could also create a separate form and controller just for the tab because all information is in there.

Reply:
This is used to display / or not display the form itself. 
If the form is not displayed, we delete the association/model
If the form is displayed, we save the assocation/model

<img width="243" alt="Capture d’écran 2023-05-26 à 10 52 33" src="https://github.com/openfoodfoundation/openfoodnetwork/assets/296452/87ce776b-ab6e-4955-994a-803532678da7">

I didn't think about that draft mode. 
Maybe something for another PR?

Reply:
Good call, maybe for another PR.